docs/adoption.md

```diff
@@ -60,190 +60,37 @@ An Azure service principal is an identity created that can be used for automated
-  type = string
-}
-variable "container_name" {
-  type = string
-}
-variable "resource_group_name" {
-  type = string
-}
-variable "resource_group_location" {
-  type = string
-}
-variable "key" {
-  type = string
-}
-```
-
-_Note: You should know the value of all these variables, apart from key which will be explained further along the tutorial._
+### Terraform state storage
 
-### Adding GitHub actions job and secrets
-
-Once you have added a remote backend to your terraform and created a service principal, GitHub actions should be configured to deploy resources to azure using terraform. We will be using the [setup-terraform](https://github.com/hashicorp/setup-terraform) action to run terraform in github actions
-
-1. Add the following secrets as your github repository secrets (Note: navigate to your Service principal under Active Directory → App registrations → select your app registration and navigate to overview):
-
-   `AZURE_AD_CLIENT_SECRET = value is in your clipboard from Step 2.4`
-   `AZURE_AD_CLIENT_ID = This is the Application (client) ID`
-   `AZURE_AD_TENANT_ID = This is the Directory (tenant) ID`
-   `AZURE_SUBSCRIPTION_ID = Navigate to subscriptions and select the Subscription ID for your subscription `
-
-2. You will then reference this as environment variables in your github actions workflow. There will be an example provided further down which you can replicate. This allows the setup-terraform action to use the service principal credentials to provision your resources.
-
-3. Add STORAGE_ACCOUNT_KEY as a repository secret. This is the storage account key for your terraform backend. You can obtain this value if you navigate to Storage accounts, select the storage account for your terraform backend, select Access Keys, click on show keys and copy the top key value. (Note: These are rotating keys and are subject to change, this tutorial does not investigate how to work around this)
-
-4. Below is an example of how your workflow should look:
+Terraform is an infrastructure as code tool which we will be using to provision the Azure resources to deploy our application. Terraform needs to be able to store its state so it is aware of what resources it has created/destroyed/updated etc. This state will be stored remotely in an Azure storage container. Complete the following steps in the Azure web portal to create the storage container:
 
-```
-name: "Terraform"
+1. Create a resource group for your infrastructure or use an existing one where your resources can be deployed. Add the resource group name and location as `RESOURCE_GROUP_NAME` and `RESOURCE_GROUP_LOCATION` respectively to github actions secrets.
+2. Create a storage account, and attach it to the resource group you created above. Add the name of the storage account you created to github secrets as `STORAGE_ACCOUNT_NAME`. Add `STORAGE_ACCOUNT_KEY` with value `prod.terraform.tfstate` to github secrets, this will be name name of the file where the state is sotred
```

Comment:
Will `STORAGE_ACCOUNT_KEY` always have a value of `prod.terraform.tfstate`?
This value contains `prod` and I wonder if there would also be a value with `staging` (or equivalent).